projects/js-packages/components/components/record-meter-bar/index.tsx

```diff
@@ -106,6 +107,24 @@ const RecordMeterBar: React.FC< RecordMeterBarProps > = ( {
 					} ) }
 				</ul>
 			</div>
+			{ /* <table className="screen-reader-text"> */ }
+			<table>
+				<caption>{ __( 'Summary of your records', 'jetpack' ) }</caption>
```

Comment:
May be we can remove "your" here?

```suggestion
				<caption>{ __( 'Summary of the records', 'jetpack' ) }</caption>
```